internal/admin/exports/form.go

```diff
@@ -38,6 +39,19 @@ type formData struct {
 	SigInfoIDs       []int64       `form:"siginfo"`
 }
 
+// splitRegions turns a string of regions (generally separated by newlines), and
+// breaks them up into a slice of strings.
+func splitRegions(regions string) []string {
+	ret := make([]string, 0)
```

Comment:
This should probably have some sane starting capacity.

Reply:
And should we do any kind of sorting for consistency?

Reply:
Sure, NBD.